### DISCUSSION TITLE
go back to previous route

### DISCUSSION THREAD

**Discussion by ZeeshanMustfai** [gilbarbara/react-joyride#1023] [Q&A] [answered]
Hello everyone, 
For example if we have 5 different pages and every page have 4 tour points. Then whenever i go back to previous page then currentindex start from 1 element of that page instead of last one. I need to start 7 > 6 > 5 like that but now its going like 7 > 4 > 5 > 6 which is wrong.
how tp fix this issue ?

**Comment by gilbarbara** [comment, marked as answer]
Hey @ZeeshanMustfai 

It's impossible to know without seeing the code, but you should probably consider this scenario in your callback.
If the `ACTION` is **prev**, subtract from the currentIndex instead.
